pythonforandroid/recipes/python2/patches/fix-init-site.patch

```diff
@@ -0,0 +1,17 @@
+--- Python-2.7.2.orig/Python/pythonrun.c	2010-10-29 05:45:34.000000000 +0200
++++ Python-2.7.2/Python/pythonrun.c	2011-04-20 17:52:12.000000000 +0200
+@@ -254,9 +254,13 @@
+     _PyGILState_Init(interp, tstate);
+ #endif /* WITH_THREAD */
+ 
++    /* For PGS4A, we don't want to call initsite, as we won't have the
```

Comment:
Can we add a proper `#define` flag here as well? Again to make it more suitable for a potential upstream merge. Something like `#ifdef NOINITSITE` or similar.